api_protocol_parameters.go

```diff
@@ -34,24 +38,28 @@ type basicProtocolParameters struct {
 	// PunishmentEpochs is the number of epochs worth of Mana that a node is punished with for each additional validation block it issues.
 	PunishmentEpochs EpochIndex `serix:"12,mapKey=punishmentEpochs"`
 
-	// LivenessThreshold is used by tip-selection to determine the if a block is eligible by evaluating issuingTimes
+	// LivenessThresholdLowerBound is used by tip-selection to determine if a block is eligible by evaluating issuingTimes.
+	// and commitments in its past-cone to ATT and lastCommittedSlot respectively.
+	LivenessThresholdLowerBound time.Duration `serix:"13,mapKey=livenessThresholdLowerBound"`
+	// LivenessThresholdUpperBound is used by tip-selection to determine if a block is eligible by evaluating issuingTimes
 	// and commitments in its past-cone to ATT and lastCommittedSlot respectively.
-	LivenessThreshold SlotIndex `serix:"13,mapKey=livenessThreshold"`
+	LivenessThresholdUpperBound time.Duration `serix:"14,mapKey=livenessThresholdUpperBound"`
```

Comment:
by using `time.Duration` the value in the protocol parameters is nanoseconds, so 15 seconds would be 15,000,000,000